src/main/app/src/assets/i18n/nl.json

```
@@ -721,7 +721,7 @@
   "msg.error.klanten.client.exception": "Er heeft zich helaas een technische fout voor gedaan. Er kan geen klantinformatie worden opgehaald.",
   "msg.error.objects.client.exception": "Er heeft zich helaas een technische fout voor gedaan. Er kunnen geen objecten worden opgehaald.",
   "msg.error.objecttypes.client.exception": "Er heeft zich helaas een technische fout voor gedaan. Er kunnen geen objecttypes worden opgehaald.",
-  "msg.error.productaanvraagtype.already.in.use": "Dit productaanvraagtype is al gekoppeld aan een ander zaaktype en kan daarom niet gebruikt worden.",
+  "msg.error.productaanvraagtype.already.in.use": "Ingevoerde productaanvraagtype is al gekoppeld aan een ander zaaktype en kan daarom niet gebruikt worden.",
```

```suggestion
  "msg.error.productaanvraagtype.already.in.use": "Dit productaanvraagtype is al gekoppeld aan een ander zaaktype en kan daarom niet gebruikt worden.",
```

I aligned this exact error message with Dimpact so better leave it. Also we don't use the word `Ingevoerde` anywhere else.

---

Reverted ✅